demo/scripts/controls/contextMenu/ImageEditPlugin.ts

```diff
@@ -0,0 +1,95 @@
+import { ContextMenuItem } from './ContextMenuProvider';
+import { safeInstanceOf } from 'roosterjs-editor-dom';
+import {
+    canRegenerateImage,
+    ImageEdit,
+    resizeByPercentage,
+    resetImage,
+} from 'roosterjs-editor-plugins/lib/ImageEdit';
+import {
+    ContextMenuProvider,
+    ExperimentalFeatures,
+    IEditor,
+    ImageEditOperation,
+} from 'roosterjs-editor-types';
+
+export default class ImageEditPlugin extends ImageEdit
+    implements ContextMenuProvider<ContextMenuItem> {
+    private allowCrop: boolean;
+
+    initialize(editor: IEditor) {
+        super.initialize(editor);
+        this.allowCrop = editor.isFeatureEnabled(ExperimentalFeatures.ImageCrop);
+    }
+
+    getContextMenuItems(node: Node) {
+        const items: ContextMenuItem[] = [];
+        if (safeInstanceOf(node, 'HTMLImageElement')) {
```

Comment:
should we validate if the image is editable with the `isContentEditable` attribute before showing the context menu?